data_structures/binary_tree/basic_binary_tree.py

```diff
@@ -30,6 +30,22 @@ def display(tree: Node | None) -> None:  # In Order traversal of the tree
         display(tree.right)
 
 
+def display_using_in_order_traversal(root: Node | None, level: int = 0) -> None:
+    """
+    >>> root = Node(1)
+    >>> root.left = Node(0)
+    >>> root.right = Node(2)
+    >>> display_using_in_order_traversal(root)
+        2
+    1
+        0
+    """
```

Comment:
If I turn this diagram 90 clockwise (so that 1 is on top) then 2 is to the left of 1 and 0 is to the right of 1 which is the opposite of lines 36 and 37.